src/include/traffic_cop/traffic_cop.h

```diff
@@ -282,6 +289,7 @@ class TrafficCop {
   common::ManagedPointer<optimizer::StatsStorage> stats_storage_;
   uint64_t optimizer_timeout_;
   const bool use_query_cache_;
+  uint64_t query_cache_timestamp_;
```

Comment:
Should this be a strong typedef timestamp_t?

Reply:
I realize this is actually metrics timestamps and not txn::timestamps. They have no relationship I guess?